File: exercises/csharp/TriviaGame/TriviaGame/Question.cs
namespace TriviaGame
{
    public class Question
    {
        public string category { get; set; }
        public int index { get; set; }
        public string text { get; set; }

        public Question(string category, int index)
        {
            this.category = category;
            this.index = index;
            text = CreateQuestionText();
        }

        private string CreateQuestionText()
        {
            return $"{category} Question " + index;
        }
    }
}

File: exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Question.java
package com.adaptionsoft.games.uglytrivia;

public class Question {
    private final String category;
    private final int index;

    public Question(String category, int index) {
        this.category = category;
        this.index = index;
    }

    @Override
    public String toString() {
        return category + " Question " + index;
    }
}
